build/tasks/build.js

```diff
@@ -13,6 +13,14 @@ gulp.task('build-html', function () {
     .pipe(gulp.dest(paths.output + 'system'));
 });
 
+gulp.task('build-css', function () {
```

Comment:
I see, the problem is the library already supports Bootstrap 4, if we want to add this we need to also provide the implementation for Bootstrap 4...